### PR TITLE
Streamtape additional domain

### PR DESCRIPTION
**Pull request by host505**

### CODE REVIEW

**Comment by host505** [comment]
Sorry, wrong pattern regex. Can you help, I'm not good at it.
@Gujal00 or @shellc0de27

**Comment by jewbmx** [comment]
I use what jsergio tossed me a while back... https://regex101.com/r/6sjcce/1
Page should load to a perfect example of what your doin too 😉

File: lib/resolveurl/plugins/streamtape.py [hunk 1]
```diff
@@ -25,8 +25,8 @@
 
 class StreamTapeResolver(ResolveUrl):
     name = "streamtape"
-    domains = ['streamtape.com', 'strtape.cloud', 'streamtape.net', 'streamta.pe', 'streamtape.site']
-    pattern = r'(?://|\.)(str(?:eam)?tap?e?\.(?:com|cloud|net|pe|site))/(?:e|v)/([0-9a-zA-Z]+)'
+    domains = ['streamtape.com', 'strtape.cloud', 'streamtape.net', 'streamta.pe', 'streamtape.site', 'strcloud.link']
+    pattern = r'(?://|\.)(str(?:eam|cloud)?tap?e?\.(?:com|cloud|net|pe|site|link))/(?:e|v)/([0-9a-zA-Z]+)'
```
Comment (Gujal00) [inline lib/resolveurl/plugins/streamtape.py]: Please update the regex to the following and test
```python
pattern = r'(?://|\.)(str(?:eam)?(?:tap?e?|cloud)\.(?:com|cloud|net|pe|site|link))/(?:e|v)/([0-9a-zA-Z]+)'
```

**Review by Gujal00 — CHANGES_REQUESTED** [review]

File: lib/resolveurl/plugins/streamtape.py (hunk 1 above)
Comment (host505) [reply lib/resolveurl/plugins/streamtape.py]: Done, thx

**Review by Gujal00 — APPROVED** [review]

**Comment by Gujal00** [comment]
Thanks

**Comment by Gujal00** [comment]
@host505 can you please start using `https://github.com/Gujal00/ResolveURL`
I wont be looking into this repo anymore. Thanks

**Pull request merged**